tools/aie-opt/CMakeLists.txt

```diff
@@ -10,13 +10,13 @@ llvm_map_components_to_libnames(llvm_libs support core irreader)
 # Now build our tools
 add_llvm_executable(aie-opt aie-opt.cpp)
 llvm_update_compile_flags(aie-opt)
-add_dependencies(aie-opt 
-  MLIRADFIncGen
-  MLIRAIEIncGen
-  MLIRAIEEnumsIncGen
-  MLIRAIEPassIncGen
-  MLIRAIEVecPassIncGen
-)
+# add_dependencies(aie-opt 
+#   MLIRADFIncGen
+#   MLIRAIEIncGen
+#   MLIRAIEEnumsIncGen
+#   MLIRAIEPassIncGen
+#   MLIRAIEVecPassIncGen
+# )
```

Comment:
Is there a reason to leave them as comments? Otherwise, I think it's preferable to just delete those lines.